### DISCUSSION TITLE
GWT support

### DISCUSSION THREAD

**Discussion by hpehl** [patternfly-java/patternfly-java#145] [Polls]
Currently PatternFly Java supports J2CL only. How important is it for you that PatternFly Java supports both J2CL and GWT?

**Comment by FrankHossfeld** [comment]
I do not think, that J2CL will soon replace GWT completely. So, a GWT version of PatternFly would be nice. 

Depending on the amount of work to get it working, it has a value.

**Comment by hpehl** [comment]
I filed #146 to address this.
